sklearn/ensemble/tests/test_gradient_boosting_loss_functions.py

```diff
@@ -288,7 +289,7 @@ def test_init_raw_predictions_values():
         raw_predictions = loss.get_init_raw_predictions(y, init_estimator)
         for k in range(n_classes):
             p = (y == k).mean()
-        assert_almost_equal(raw_predictions[:, k], np.log(p))
+            assert_allclose(raw_predictions[:, k], np.log(p))
```

Comment:
With the last commit, I took the liberty of replacing `assert_almost_equal` with either `approx` or `assert_allclose`.

Note the additional indentation of this line!